devito/finite_differences/differentiable.py

```diff
@@ -240,16 +240,17 @@ def laplace(self):
         derivs = tuple('d%s2' % d.name for d in space_dims)
         return Add(*[getattr(self, d) for d in derivs])
 
-    @property
-    def div(self):
+    def div(self, shift=None):
         space_dims = [d for d in self.dimensions if d.is_Space]
-        derivs = tuple('d%s' % d.name for d in space_dims)
-        return Add(*[getattr(self, d) for d in derivs])
+        return Add(*[getattr(self, 'd%s' % d.name)(x0=None if shift is None else
```

Comment:
easier to read as

```
x0=shift or (d + shift * d.spacing)
```

?

Reply:
That's also not the same and wrong.

`None if shift is None else  (d + shift * d.spacing)` 
- None => None
- value => d + value * spacing

`x0=shift or (d + shift * d.spacing)` : 
- None => d + None * d.spacing , that's not valid
- value => value

Reply:
ah, LOL, don't know what my mind was processing when I wrote that comment above, ignore me :grimacing: